### ISSUE TITLE
Improve responsive layout for narrower screens

### ISSUE THREAD

**Issue by leahbannon** [18F/federalist#424]
From the UX study:

On smaller screens, Edit content gets pushed down. When participants’ browser window is not wide enough, the side nav (Pages, Media, Settings, Logs) pushes the Edit content down, making it difficult for them to use the image editor and text editor at the same time.

Recommendation 1: Consider making the left nav responsive (nav items appear horizontally next to each other, instead of vertically) so that the Edit content doesn’t pushed down more than it has to.

Recommendation 2: Image preview thumbnails are taller than they need to be. Consider making the thumbnail area fit to the image that has been uploaded. 

<img width="352" alt="screen shot 2016-06-22 at 6 00 14 pm" src="https://cloud.githubusercontent.com/assets/4323988/16288668/3202f8f0-38a3-11e6-8bdb-bff99f3e7c1e.png">

**Comment by wslack** [comment]
Closing as the editor has been disabled.
